js/lib/communityCall.json

```diff
@@ -0,0 +1,6 @@
+{
+    "callName": "IPFS Weekly Call",
+    "time": "17:00 UTC",
+    "day": "Monday",
+    "zoomLink": "https://protocol.zoom.us/j/443621844"
```

Comment:
Given that the banner is going to be displayed 2 hours before the call is it better to link them to a page with more info about the call, which has the notes link as well as the zoom link? Otherwise people might be sat in an empty room for 2 hours.

Or link directly to the zoom when the call is happening, but to a "more information" page before?

Reply:
> Given that the banner is going to be displayed 2 hours before the call is it better to link them to a page with more info about the call, which has the notes link as well as the zoom link?Otherwise people might be sat in an empty room for 2 hours.

Yes, it would be better to link people to a "more information" page. Will swap out link for "more information page" https://github.com/ipfs/team-mgmt/issues/549